lib/internal/streams/operators.js

```diff
@@ -219,6 +219,7 @@ function asIndexedPairs(options = undefined) {
 
 async function some(fn, options = undefined) {
   for await (const unused of filter.call(this, fn, options)) {
+    this.destroy(null);
```

Comment:
should I use `destroyImpl.destroyer`? because if so the abort error will still be thrown